subsys/dfu/dfu_target/src/dfu_target_mcuboot.c

```diff
@@ -56,7 +56,7 @@ int dfu_ctx_mcuboot_set_b1_file(const char *file, bool s0_active,
 	}
 
 	/* We have verified that there is a null-terminator, so this is safe */
-	char *space = strstr(file, " ");
+	char *space = strstr(file, "+");
```

Comment:
rename to delimiter. Space does not make sens in the context anymore. Needs to be changed below also.
```suggestion
char *delimiter = strstr(file, "+");

Reply:
Updated as per advised